src/service/auth.service.ts

```diff
@@ -0,0 +1,37 @@
+import { Injectable } from '@nestjs/common';
+import { JwtService } from '@nestjs/jwt';
+
+@Injectable()
+export class AuthService {
+
+    constructor(private readonly jwtService: JwtService) { }
+
+    async createAccessToken(userId: string, roles: string[]): Promise<string> {
+
+        const payload = { userId, roles };
+        return this.jwtService.sign(payload);
+
+    }
+
+    async createRefreshToken(userId: string): Promise<string> {
+
+        const payload = { userId };
```

Comment:
why do you send RefreshToken && AccessToken for random user who provide  userId ??